File: Backend/tiendaARK/src/main/java/cl/difrap/productos/tiendaark/dto/Item.java
package cl.difrap.productos.tiendaark.dto;

import cl.difrap.biblioteca.Entidad;

public class Item extends Entidad
{
	private String nombre;
	private Long cantidad;
	private String comando;
	private Long precio;
	private String icono;
	private Integer tipo;
	
	public Item() 
	{
		super(null);
	}

	public String getNombre() {
		return nombre;
	}

	public void setNombre(String nombre) {
		this.nombre = nombre;
	}

	public Long getCantidad() {
		return cantidad;
	}

	public void setCantidad(Long cantidad) {
		this.cantidad = cantidad;
	}

	public String getComando() {
		return comando;
	}

	public void setComando(String comando) {
		this.comando = comando;
	}

	public Long getPrecio() {
		return precio;
	}

	public void setPrecio(Long precio) {
		this.precio = precio;
	}

	public String getIcono() {
		return icono;
	}

	public void setIcono(String icono) {
		this.icono = icono;
	}

	public Integer getTipo() {
		return tipo;
	}

	public void setTipo(Integer tipo) {
		this.tipo = tipo;
	}
	
	
	
}

File: Frontend/tienda-ark/src/app/models/item.ts
import { Entidad,IEntidad } from "./entidad";

interface IItem extends IEntidad
{
    nombre?:String,
    cantidad?:Number,
    comando?:String,
    precio?:Number,
    icono?:String,
    tipo?:Number
}
export class Item extends Entidad
{
    public idIncremental:any=null;
    public nombre:any=null;
    public cantidad:any=null;
    public comando:any=null;
    public precio:any=null;
    public icono:any=null;
    public tipo:any=null;
    constructor(entidad?:IItem)
    {
        super();
        Object.assign(this, entidad);
    }
}